### PR TITLE
feat: fallback to default browser if Chrome is not found

### PR DESCRIPTION
**Pull request by thecodrr**
Summary:
---------

It can be annoying for users (like me) who do not have Google Chrome installed to enable 'Remote Debugging' only to receive an error. This rectifies that by falling back to the default installed browser making the whole experience more fluid.

Test Plan:
----------
Tested on a sample app by enabling 'Remote Debugging'.

### CODE REVIEW

File: packages/cli/src/commands/server/launchDebugger.js [hunk 1]
```diff
@@ -24,6 +26,28 @@ function commandExistsUnixSync(commandName) {
   }
 }
 
+function commandExistsWindowsSync(commandName) {
+  try {
+    var stdout = execSync('where ' + commandName, {stdio: []});
```
Comment (thymikee) [inline packages/cli/src/commands/server/launchDebugger.js]:
```suggestion
    const stdout = execSync('where ' + commandName, {stdio: []});
```

Comment (theweavrs) [reply packages/cli/src/commands/server/launchDebugger.js]: Fixed.

File: packages/cli/src/commands/server/middleware/getDevToolsMiddleware.js [hunk 2]
```diff
@@ -6,29 +6,29 @@
  *
  * @format
  */
-import launchChrome from '../launchChrome';
 import {logger} from '@react-native-community/cli-tools';
 import {exec} from 'child_process';
+import launchDebugger from './../launchDebugger';
```
Comment (thymikee) [inline packages/cli/src/commands/server/middleware/getDevToolsMiddleware.js]:
```suggestion
import launchDebugger from '../launchDebugger';
```

Comment (theweavrs) [reply packages/cli/src/commands/server/middleware/getDevToolsMiddleware.js]: I don't see why it shouldn't but nothing wrong in testing. I use Brave and it works without an issue but that uses Chrome DevTools. I will test this on Firefox etc. These are essentially websockets and http requests so in theory it should work

File: packages/cli/src/commands/server/launchDebugger.js [hunk 3]
```diff
@@ -52,4 +76,18 @@ function launchChrome(url: string) {
...
 
-export default launchChrome;
+function launchDebugger(url: string) {
+  if (!commandExists(getChromeAppName())) {
+    logger.info(
+      `For a better debugging experience please install Google Chrome from: ${chalk.underline.dim(
+        'https://www.google.com/chrome/',
+      )}`,
+    );
+    //fallback to default browser
```
Comment (grabbou) [inline packages/cli/src/commands/server/launchDebugger.js]: nit: space here before the inline command. I also think we can rid of that comment and rename the function to something like: `launchDefaultBrowser`

Comment (theweavrs) [reply packages/cli/src/commands/server/launchDebugger.js]: Done.

**Pull request merged**